docs/requirements.txt

```diff
@@ -1,2 +0,0 @@
-sphinx
-sphinx_rtd_theme
```

Comment:
what's the purpose of this?

Reply:
Vestigial file which should've been deleted in #1544.